domain/model/state/modelstate.go

```diff
@@ -216,21 +231,14 @@ FROM model
 INSERT INTO model (*) VALUES ($dbReadOnlyModel.*)
 `, dbReadOnlyModel{})
 	if err != nil {
-		return errors.Annotatef(err, "preparing insert model statement")
+		return errors.Capture(err)
 	}
 
-	var outcome sqlair.Outcome
-	if err := tx.Query(ctx, insertStmt, m).Get(&outcome); err != nil {
-		return fmt.Errorf("creating model %q: %w", args.UUID, err)
+	if err := tx.Query(ctx, insertStmt, m).Run(); err != nil {
+		return errors.Errorf(
```

Comment:
Do we want to check unique constraints violation error to return an AlreadyExists error?

Reply:
Thanks, this was a good pickup. Have also added a test for this.

Reply:
This is already done above and a test exists asserting this. Have updated comment on func to reflect the contract.